packages/atomic/src/components/common/query-error/icon.spec.ts

```diff
@@ -0,0 +1,71 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {html} from 'lit';
+import {describe, it, expect} from 'vitest';
+import CannotAccess from '../../../images/cannot-access.svg';
+import Indexing from '../../../images/indexing.svg';
+import NoConnection from '../../../images/no-connection.svg';
+import SearchInactive from '../../../images/search-inactive.svg';
+import SomethingWrong from '../../../images/something-wrong.svg';
+import {renderQueryErrorIcon} from './icon';
+
+describe('#renderQueryErrorIcon', () => {
+  const renderComponent = async (errorType?: string) => {
+    const element = await renderFunctionFixture(
+      html`${renderQueryErrorIcon({props: {errorType}})}`
+    );
+
+    return {
+      atomicIcon: element.querySelector('atomic-icon'),
+    };
+  };
+
+  it('should render atomic-icon with correct attributes', async () => {
+    const {atomicIcon} = await renderComponent();
+
+    expect(atomicIcon).not.toBeNull();
+    expect(atomicIcon).toHaveAttribute('part', 'icon');
+    expect(atomicIcon).toHaveAttribute('class', 'w-1/2 max-w-lg');
+  });
+
+  it('should render default icon when errorType is undefined', async () => {
```

Comment:
I find it a bit odd that we're still rendering something when the errorType is undefined, but this was the original behavior as well